chart/BUILD.yaml

```diff
@@ -14,9 +14,9 @@ packages:
       - imageRepoBase
     config:
       commands:
-        # set values
-        - ["yq", "w", "-i", "Chart.yaml", "version", "${version}"]
-        - ["yq", "w", "-i", "values.yaml", "version", "${version}"]
+        # set values. Use the value in values.yaml:.version as ${version}
+        - ["sh", "-c", "yq w -i Chart.yaml version $(yq r values.yaml version)"]
```

Comment:
That would break with [this proposed changed](https://github.com/gitpod-io/gitpod/pull/3192) where we remove the version from the `values.yaml`, right?

Couldn't we just use a version like this one (`0.0.1613398538`)?
```
echo "0.0.$(date +%s)"
```
That should be semver compatible, right?

Or even:
```
echo "0.0.$(date +%s)-${version}"
```

Reply:
:facepalm: Ah, totally forgot about this. Yes, both fine.

Thank you!

Reply:
> That would break with this proposed changed where we remove the version from the values.yaml, right

does `yq w` not insert the `version` when it's not there? 

About the version format: I very much like to have the brach name as qualifier in the version, because I think that makes it much more understandable where this version came from. Also, for releases we like to use `-rc1`, `-beta1` etc.

Reply:
> does `yq w` not insert the `version` when it's not there?

Yes, but the problematic part is: `$(yq r values.yaml version)`

> I very much like to have the brach name as qualifier in the version, because I think that makes it much more understandable where this version came from

Probably we don't need the timestamp at all. Just
```
0.0.0-${version}
```
should be sufficient to meet the semver requirements and are close to the current situation, right?

Reply:
I went with `0.1.0-${version}`.